livingatlas/pipelines/src/main/java/au/org/ala/pipelines/interpreters/ALALocationInterpreter.java

```diff
@@ -94,124 +91,54 @@
-              lr.setStateProvince(formalStateName.get());
-            }
+          // Formalize state name
+          Optional<String> formalStateName = stateProvinceVocab.matchTerm(lr.getStateProvince());
+          if (formalStateName.isPresent()) {
+            lr.setStateProvince(formalStateName.get());
+          }
 
-            String suppliedStateProvince = extractNullAwareValue(er, DwcTerm.stateProvince);
-            if (!Strings.isNullOrEmpty(suppliedStateProvince)) {
-              // If the stateProvince that is retrieved using the coordinates differs from the
-              // supplied stateProvince
-              // raise an issue
-              Optional<String> formalSuppliedName =
-                  StateProvince.getInstance(
-                          alaConfig.getLocationInfoConfig().getStateProvinceNamesFile())
-                      .matchTerm(suppliedStateProvince);
-              if (formalSuppliedName.isPresent()) {
-                suppliedStateProvince = formalSuppliedName.get();
-              }
-              if (!suppliedStateProvince.equalsIgnoreCase(lr.getStateProvince()))
-                addIssue(lr, ALAOccurrenceIssue.STATE_COORDINATE_MISMATCH.name());
+          String suppliedStateProvince = extractNullAwareValue(er, DwcTerm.stateProvince);
+          if (!Strings.isNullOrEmpty(suppliedStateProvince)) {
+            // If the stateProvince that is retrieved using the coordinates differs from the
+            // supplied stateProvince
+            // raise an issue
+            Optional<String> formalSuppliedName =
+                stateProvinceVocab.matchTerm(suppliedStateProvince);
+            if (formalSuppliedName.isPresent()) {
+              suppliedStateProvince = formalSuppliedName.get();
             }
+            if (!suppliedStateProvince.equalsIgnoreCase(lr.getStateProvince()))
+              addIssue(lr, ALAOccurrenceIssue.STATE_COORDINATE_MISMATCH.name());
+          }
 
-            if (StateProvinceCentrePoints.getInstance(alaConfig.getLocationInfoConfig())
-                .coordinatesMatchCentre(
-                    lr.getStateProvince(), lr.getDecimalLatitude(), lr.getDecimalLongitude())) {
-              addIssue(lr, ALAOccurrenceIssue.COORDINATES_CENTRE_OF_STATEPROVINCE.name());
-            } else {
-              if (log.isTraceEnabled()) {
-                log.trace(
-                    "{},{} is not the centre of {}!",
-                    lr.getDecimalLatitude(),
-                    lr.getDecimalLongitude(),
-                    lr.getStateProvince());
-              }
+          if (stateProvinceCentrePoints.coordinatesMatchCentre(
+              lr.getStateProvince(), lr.getDecimalLatitude(), lr.getDecimalLongitude())) {
+            addIssue(lr, ALAOccurrenceIssue.COORDINATES_CENTRE_OF_STATEPROVINCE.name());
+          } else {
+            if (log.isTraceEnabled()) {
+              log.trace(
+                  "{},{} is not the centre of {}!",
```

Comment:
Has this got a typo? Should it read "...is [at] the centre of {}" ?

Reply:
it has yes - good catch

Reply:
i.e. no issue needs to be raise....

Reply:
Indeed. I am blind